Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java

```diff
@@ -900,6 +900,47 @@ private void updateDataSourceStringValue(CorrelationDataSource eamDataSource, St
         }
     }
 
+    /**
+     * Changes the name of a data source in the DB
+     * 
+     * @param eamDataSource  The data source
+     * @param newName        The new name
+     * 
+     * @throws EamDbException 
+     */
+    @Override
+    public void updateDataSourceName(CorrelationDataSource eamDataSource, String newName) throws EamDbException {
+        
+        Connection conn = connect();
+
+        PreparedStatement preparedStatement = null;
+
+        String sql = "UPDATE data_sources SET name = ? WHERE id = ?";
+
+        try {
+            preparedStatement = conn.prepareStatement(sql);
+            preparedStatement.setString(1, newName);
+            preparedStatement.setInt(2, eamDataSource.getID());
+            preparedStatement.executeUpdate();
+            
+            CorrelationDataSource updatedDataSource = new CorrelationDataSource(
```

Comment:
Merged changes appear to have caused there to no longer be a constructor with these arguments.